### PR TITLE
Update service generator to use Prisma namespace

### PR DESCRIPTION
**Pull request by o0charlie0o**
Prisma placed all of their types behind the Prisma namespace. The service generator is still using the old implementation in the template. This PR fixes that.

The following issue discusses this but it doesn't actually seem to be resolved:
- https://github.com/redwoodjs/redwood/issues/1534

### CODE REVIEW

**Comment by thedavidprice** [comment]
@o0charlie0o thanks for this! Is the PR ready for review? Assuming not since it's a draft. Do let me know and I’ll loop in someone for review.

**Comment by o0charlie0o** [comment]
@thedavidprice it's ready now. Just had to update the tests. Thanks!

File: packages/cli/src/commands/generate/service/__tests__/fixtures/multiWord_crud.ts [hunk 1]
```diff
@@ -1,22 +1,18 @@
-import {
-  UserProfileWhereUniqueInput,
-  UserProfileCreateInput,
-  UserProfileUpdateInput,
-} from '@prisma/client'
+import { Prisma } from '@prisma/client'
```
Comment (dac09) [inline packages/cli/src/commands/generate/service/__tests__/fixtures/multiWord_crud.ts:1]: Should this be the default export, so like:
```
import Prisma from '@prisma/client'
```

Question, not a suggestion :)

Comment (o0charlie0o) [reply packages/cli/src/commands/generate/service/__tests__/fixtures/multiWord_crud.ts:1]: @dac09 The Prisma docs specify `import { Prisma } from '@prisma/client'`: https://www.prisma.io/docs/concepts/components/prisma-client/working-with-generated-types

Also the Prisma code mod updates all of the type imports to this syntax as well:
https://github.com/prisma/codemods

I just tested in my codebase and the Prisma namespace is only available if you were to do something like `Prisma.Prisma.ModelWhereUniqueInput`.

**Review by dac09 — APPROVED** [review]

**Comment by cannikin** [comment]
I think this looks okay? But it's Typescript so... 🤷‍♂️

**Pull request merged**